Gems/Atom/RPI/Code/CMakeLists.txt

```diff
@@ -17,6 +17,8 @@ else()
     set(MASKED_OCCLUSION_CULLING_FILES "")
 endif()
 
+include(3rdParty/FetchSPD.cmake)
```

Comment:
Is this including a file from the github src that is copied at build time into the source original folders of the gem? It should instead produce a library and link using BUILD_DEPENDENCIES. See Recast Navigation example in my other comment.

Reply:
This file should be changed to produce INTERFACE target should be produced, that the Atom_RPI depends on

Reply:
Thanks but I removed FetchSPD.cmake.